html/elements/summary.json

```diff
@@ -62,7 +62,7 @@
               },
               "edge": {
                 "version_added": false,
-                "notes": "Edge currently doesn't use <code>display: list-item</code> on the <code>&lt;summary&gt;</code> element, so <code>display: block</code> will not automatically hide the disclosure widget. Instead, use the non-standard pseudo-element <code>::-webkit-details-marker</code> to change the disclosure widget in Edge. See <a href='https://crbug.com/590014'>Edge bug 590014</a> for details."
+                "notes": "Edge currently doesn't use <code>display: list-item</code> on the <code>&lt;summary&gt;</code> element, so <code>display: block</code> will not automatically hide the disclosure widget. Instead, use the non-standard pseudo-element <code>::-webkit-details-marker</code> to change the disclosure widget in Edge. See <a href='https://crbug.com/590014'>Chrome bug 590014</a> for details."
```

Comment:
Nice find, Vinyl. I don't think it adds anything to say that it's an Edge or Chrome bug. Really it's both, or neither—the more correct thing would be to say "Chromium". But given the context, there's no need to specify at all:

```suggestion
                "notes": "Edge currently doesn't use <code>display: list-item</code> on the <code>&lt;summary&gt;</code> element, so <code>display: block</code> will not automatically hide the disclosure widget. Instead, use the non-standard pseudo-element <code>::-webkit-details-marker</code> to change the disclosure widget in Edge. See <a href='https://crbug.com/590014'>bug 590014</a> for details."
```